<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.3">
    <title>Links for django-creta-auth</title>
  </head>
  <body>
    <h1>Links for django-creta-auth</h1>
<a href="https://files.pythonhosted.org/packages/5e/c4/ade5b1e97710df0dd75ab8e34e9b1cab606e78818f8bcec3940a7f6de29f/django-creta-auth-0.1.0.tar.gz#sha256=ab45955c3660a24888a7d96bb3d22ccadfad56cd0f1a46aa81bb7b3bc8e12c87" data-requires-python="&gt;=3.6" >django-creta-auth-0.1.0.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/c4/64/c9c3670fcbb1645f94523e2c87a2281c8be4d638e8727f8549ce2c6f131c/django-creta-auth-0.1.3.tar.gz#sha256=e5fcc2cc15a7457bcb8cd2db4508ab337648bd6bceb4a97787b826d0184d52be" data-requires-python="&gt;=3.6" >django-creta-auth-0.1.3.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/63/fd/fb12eb8a5f8c38fa485cffc9613e27463ad165ca97a8f208fab55ebb10df/django-creta-auth-0.1.4.tar.gz#sha256=ac024c7f90abba062980375e3988d6503277549ec8793579b180b56a19665662" data-requires-python="&gt;=3.6" >django-creta-auth-0.1.4.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/ab/24/016521f2f53034ce7f4ffe9dafa3de1d204b9666434825d276c077f7a579/django_creta_auth-0.1.4-py3-none-any.whl#sha256=834c75c26d5f49621f8b90b8a898500df6854852730cb36774f03e25d3735aa0" data-requires-python="&gt;=3.6" data-dist-info-metadata="sha256=64f27846e37be98093042e0846aa312f7a8888b83849344520559ca18b41b9dd" data-core-metadata="sha256=64f27846e37be98093042e0846aa312f7a8888b83849344520559ca18b41b9dd">django_creta_auth-0.1.4-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/65/3a/f7f2892d5eb834b6120cbf0e82bcde0ea2f01f690abe5897bb4d7df53b65/django-creta-auth-0.1.5.tar.gz#sha256=77bf62622ed640766e53fa793a1e1a04002b5ad914ea12603c8c30e6aa5fa76d" data-requires-python="&gt;=3.6" >django-creta-auth-0.1.5.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/e6/cd/0b14a6d3762a6de59a665dab601296764885a3d02e713ae899a8816d2ee3/django_creta_auth-0.1.5-py3-none-any.whl#sha256=c0781f4ade4999cd9052f7f9988cf9f4840c845721b333763f8dd5c2fa23d0c6" data-requires-python="&gt;=3.6" data-dist-info-metadata="sha256=00d8f32f264054650d4a966f4d8f15db81a0b114871eadf2ccd07f27faf6d525" data-core-metadata="sha256=00d8f32f264054650d4a966f4d8f15db81a0b114871eadf2ccd07f27faf6d525">django_creta_auth-0.1.5-py3-none-any.whl</a><br />
</body>
</html>
<!--SERIAL 23980134-->